__tests__/app.test.js

```diff
@@ -278,3 +278,57 @@ describe("task 7. POST /api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("Task 8 /api/articles/:article_id, update an article by id", () => {
+  test("PATCH 201, should inc vote property by 1", () => {
+    const changesToArticle = { inc_votes: 5 };
+    const output = {
+      title: "Living in the shadow of a great man",
+      topic: "mitch",
+      author: "butter_bridge",
+      body: "I find this existence challenging",
+
+      votes: 105,
+      article_img_url:
+        "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+    };
+    return request(app)
+      .patch("/api/articles/1")
+      .send(changesToArticle)
+      .expect(201)
+      .then(({ body: { article } }) => {
+        expect(article).toMatchObject(output);
+        expect(article.votes).toBe(output.votes);
```

Comment:
this is unnecessary as you are already checking the votes implicitly in the toMatchObject statement